dm/config/subtask.go

```diff
@@ -450,6 +451,14 @@ func (c *SubTaskConfig) Adjust(verifyDecryptPassword bool) error {
 	if c.SyncerConfig.CheckpointFlushInterval == 0 {
 		c.SyncerConfig.CheckpointFlushInterval = defaultCheckpointFlushInterval
 	}
+	if c.SyncerConfig.SafeModeDuration == "" {
+		c.SyncerConfig.SafeModeDuration = defaultSafeModeDuration
```

Comment:
when user has set non-empty CheckpointFlushInterval and empty SafeModeDuration, we should set SafeModeDuration to 2*CheckpointFlushInterval rather than defaultSafeModeDuration